CleverFerret/src/main/java/com/universalmedialibrary/services/metadata/covers/AmazonCoverSource.kt

```
@@ -0,0 +1,135 @@
+                .replace(Regex("\\._AC_[A-Z0-9]+_"), "._AC_SX2000_")
+                .replace(Regex("\\._SX\\d+_"), "._SX2000_")
+                .replace(Regex("\\._SY\\d+_"), "._SY2000_")
+            
+            return CoverResult(
+                url = hdUrl,
+                width = 1600,
+                height = 2400,
+                quality = CoverQuality.ULTRA_HD,
+                source = "$sourceName ($domain)"
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)